pkg/controller/operators/olm/apiservices.go

```diff
@@ -486,3 +486,34 @@ func (a *Operator) updateDeploymentSpecsWithApiServiceData(csv *v1alpha1.Cluster
 	}
 	return strategyDetailsDeployment, nil
 }
+
+func (a *Operator) areWebhooksAvailable(csv *v1alpha1.ClusterServiceVersion) (bool, error) {
```

Comment:
Won't this be an issue if there's an update to a webhook that changes the name of the webhook? I think we'll be left with an "old" webhook, right?

Reply:
Yep - I'll update the PR to handle this case and make sure that the added e2e test checks for this.

Reply:
Updated in the latest version of the PR.